packages/xo-web/src/common/intl/messages.js

```diff
@@ -542,17 +542,17 @@ const messages = {
   remoteAuth: 'Auth',
   remoteDeleteTip: 'Delete',
   remoteDeleteSelected: 'Delete selected remotes',
-  remoteMyNamePlaceHolder: 'Name *',
+  remoteMyNamePlaceHolder: 'Name',
```

Comment:
Can you please explain these changes.

Reply:
It was confusing in the `host`'s placeholder (could be interpreted as part of the pattern) and I decided to homogenize and remove it everywhere since I don't think they're very useful there anyway.